AWS.md

```diff
@@ -38,7 +38,20 @@ EC2
 
 VPC
 ---
-* Ensure you have added the correct [Gateway Endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/vpce-gateway.html) for the AWS services being accessed from your private subnets to avoid incurring unnecessary networking costs.
+
+* Avoid using the default VPC - see [here](https://konekti.us/post/are-you-using-the-aws-default-vpc-stop-now/#:~:text=The%20default%20VPC%20lacks%20the,for%20auditing%20and%20troubleshooting%20purposes.) for a list of reasons why this is a bad idea. 
+* Make sure the teams consider whether the account will need to be allocated a block of our IP address space to support peering (see [here](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-basics.html) for more info on AWS peering rules.
+* If it is likely that AWS resources will need to communicate with our on-prem infrastructure, then contact the networking team to request a CIDR allocation for the VPC.
+* If your project uses [guardian/cdk](https://github.com/guardian/cdk) to manage environments and resources, define the VPC using the CDK construct for VPC's [details here](https://github.com/guardian/cdk/blob/main/src/constructs/vpc/vpc.ts#L32-L59). 
+* Ensure you have added the correct [Gateway Endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/vpce-gateway.html) for the AWS services being accessed from your private subnets to avoid incurring unnecessary networking costs. 
+* For general-purposes, there should be one VPC per AWS account and they should have the following properties (which our VPC construct ensures/asks for in its docs):
```

Comment:
I would be tempted to lead on this in the doc. I.e. use our CDK construct. Doing the right thing and maintaining it over time without this is quite difficult as it requires expertise and also hitting a moving target (as our understanding of best practice evolves over time), whereas CDK provides all of this automatically.

E.g. something like:

---
To follow best practice for VPCs, ensure you have a single CDK-generated VPC in your account that is used to house your applications. You can find the docs for it [here](...).

For exceptional cases, you may want to use the construct to generate further VPCs to isolate specific applications. It is worth discussing with DevX Security and InfoSec if you are thinking of doing this.

You **MUST NOT** use the default VPC. It... (reasons)...

Reply:
Good idea!